lib/features/support/servlet.rb

```diff
@@ -84,8 +92,14 @@ def log_request(request)
     end
   end
 
+  # Checks the Bugsnag-Integrity header, if present, against the request and based on configuration.
+  # If the header is present, if the digest must be correct.  However, the header need only be present
+  # if configuration says so.
   def check_digest(request)
     header = request['Bugsnag-Integrity']
+    if MazeRunner.config.enforce_bugsnag_integrity
+      raise 'Bugsnag-Integrity header must be present according to MazeRunner.config.enforce_bugsnag_integrity'
+    end
```

Comment:
Won't this always raise if `enforce_bugsnag_integrity` is true?

Reply:
You're right, good spot.  This PR is merged now, but I'll create a fresh one once you've had a chance to make any further comments.

Reply:
That was my only comment 😀